src/Microsoft.Extensions.Options.ConfigurationExtensions/ConfigureFromConfigurationOptions.cs

```diff
@@ -6,7 +6,7 @@
 
 namespace Microsoft.Extensions.Options
 {
-    public class ConfigureFromConfigurationOptions<TOptions> : ConfigureOptions<TOptions>
+    public class ConfigureFromConfigurationOptions<TOptions> : ConfigureOptions<TOptions>, IOptionsConfiguration<TOptions>
```

Comment:
I'll switch IOptionsConfiguration => IOptionsSetup in the next iteration (assuming we move forward)